README.md

```diff
@@ -40,494 +40,196 @@ Ensure packages are installed with correct version numbers by running:
   npm install --save react-dates moment@>=#.## react@>=#.## react-dom@>=#.## react-addons-shallow-compare@>=#.##
   ```
 
-#### Include component
+### Include component
 ```js
-import { SingleDatePicker } from 'react-dates';
+import { DateRangePicker, SingleDatePicker, DayPickerRangeController } from 'react-dates';
 ```
 
-#### Include CSS
+### Include CSS
 When you use Webpack with CSS loader:
 ```js
 import 'react-dates/lib/css/_datepicker.css';
 ```
 Otherwise create a CSS file with the contents of `require.resolve('react-dates/lib/css/_datepicker.css')` and include it in the html `<head>` section.
 
-#### Make some awesome datepickers
-```jsx
-<SingleDatePicker
-  id="date_input"
-  date={this.state.date}
-  focused={this.state.focused}
-  onDateChange={(date) => { this.setState({ date }); }}
-  onFocusChange={({ focused }) => { this.setState({ focused }); }}
-/>
-```
-
-## API
-
-We have a handful of different components and utilities available for all of your date picking needs!
-
-### `DateRangePicker`
-This fully-controlled component is designed to allow a user to select both a start date and an end date. It is best suited for a selecting a relatively short date range some time in the next year.
-
-#### `Props`
-
-**Dates:**
+### Make some awesome datepickers
 
-Moment objects representing the currently selected start and end dates. To indicate that a date has not yet been selected, these are set to `null`.
-```js
-  startDate: momentPropTypes.momentObj,
-  endDate: momentPropTypes.momentObj,
-```
-
-`onDatesChange` is the callback necessary to update the date state being held in the parent component and pass that back down to the `DateRangePicker` as props. `onDatesChange` receives an object of the form
-`{ startDate: momentPropTypes.momentObj, endDate: momentPropTypes.momentObj }`
-as an argument.
-```js
-  onDatesChange: PropTypes.func,
-```
+We provide a handful of components for your use. Each of these have a handful of props required for a fully functional interactive date picker as well as a large number of props for customizing the look and feel of the inputs, calendar, and
```

Comment:
> Each of these have a handful of props required for a fully functional interactive date picker as well as a large number of props for customizing the look and feel of the inputs, calendar, and even individual days. 

It feels like a very long sentence with little added information to me (Disclaimer: I'm a really bad & lazy reader with longer sentence). Breaking up the sentences a little bit may help with readability, and make it relatable to the user. 
i.e. 
> If you supply essential props with each component, you'll get a full featured interactive date picker. With additional optional props, you can customize the look and feel of the inputs, calendar, etc..